src/otx/algo/action_classification/mmconfigs/x3d.yaml

```diff
@@ -15,14 +15,15 @@ cls_head:
 data_preprocessor:
   format_shape: NCTHW
   mean:
-    - 114.75
-    - 114.75
-    - 114.75
+    - 0.0
```

Comment:
Are these corrected by the pretrained checkpoint?

Reply:
I just sync with previous OTX1.x configs. I'll modify after fixing performance issues in action classification

Reply:
You must align it with the environment where the pretraining checkpoint was created since we we finetune from it.
I can see that it takes RGB and different channel normalization as opposed to those you changed
https://github.com/open-mmlab/mmaction2/blob/4d6c93474730cad2f25e51109adcf96824efc7a3/configs/recognition/x3d/metafile.yml#L17
https://github.com/open-mmlab/mmaction2/blob/4d6c93474730cad2f25e51109adcf96824efc7a3/configs/recognition/x3d/x3d_m_16x5x1_facebook-kinetics400-rgb.py#L1
https://github.com/open-mmlab/mmaction2/blob/4d6c93474730cad2f25e51109adcf96824efc7a3/configs/_base_/models/x3d.py#L15-L16